index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>mobile version</title>
+</head>
+
+<body>
+
+    <header class="header">
+        <nav class="navbar">
+            <a class="container">Arrow Dan</a>
+            <img class="hamburger hide-desktop" src="image/Union.png" alt="navigation icon" />
+        </nav>
```

Comment:
- [ ] 1. Kindly set the hamburger as a  button

Reply:
@[aimemalaika](https://github.com/aimemalaika) the hamburger thing is displayed as icon in figma... or maybe i did not get you kindly...